bcda/web/api.go

```diff
@@ -300,8 +303,9 @@ func validateRequest(r *http.Request) ([]string, *fhirmodels.OperationOutcome) {
 	// validate optional "_since" parameter
 	params, ok = r.URL.Query()["_since"]
 	if ok {
-		_, err := fhirutils.ParseDate(params[0])
-		if err != nil {
+		instantRegex := `([0-9]([0-9]([0-9][1-9]|[1-9]0)|[1-9]00)|[1-9]000)-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T([01][0-9]|2[0-3]):[0-5][0-9]:([0-5][0-9]|60)(\.[0-9]+)?(Z|(\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00))`
+		validSince, err := regexp.MatchString(instantRegex, params[0])
+		if err != nil || !validSince {
 			oo := responseutils.CreateOpOutcome(responseutils.Error, responseutils.Exception, responseutils.FormatErr, "Invalid date format supplied in _since parameter.  Date must be in FHIR DateTime format.")
```

Comment:
Error response indicates "Date must be in FHIR DateTime format."
Should this be updated to specify "Date must be in FHIR Instant format."?